PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1646,7 +1708,7 @@ def runMetCorAndUncFromMiniAOD(process, metType="PF",
     runMETCorrectionsAndUncertainties = RunMETCorrectionsAndUncertainties()
```

Comment:
somewhere in this method the patCaloMet (maybe other METs as well) should get the srcPFCands set to be using packedPFCandidates